gcp/apis/compute/v1alpha1/network_types_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestGCPNetworkSpec_IsSameAs(t *testing.T) {
+	trueVal := true
+	cases := map[string]struct {
+		spec   GCPNetworkSpec
+		status GCPNetworkStatus
+		result bool
+	}{
+		"FullMatch": {
+			spec: GCPNetworkSpec{
+				Description:           "some desc",
```

Comment:
Not vital, but it might be nice to make some of these testing fields `const` at the top of this file. Not a blocking change though.